src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1242,6 +1243,7 @@ def __init__(self,
         self.ssl = ssl
         self.keepalive_only = keepalive_only
         self.enable_haproxy_protocol = enable_haproxy_protocol
+        self.health_check_interval = health_check_interval.strip() if health_check_interval else None
```

Comment:
the line number is messed up in the error message, but it's this line it's complaining about
```
ceph/deployment/service_spec.py:1354:101: E501 line too long (101 > 100 characters)
```